pytket/tests/utils_test.py

```diff
@@ -22,7 +22,7 @@
 from hypothesis import HealthCheck, given, settings, strategies
 from hypothesis.strategies._internal import SearchStrategy
 from simulator import TketSimBackend, TketSimShotBackend  # type: ignore
-from sympy import symbols
+from sympy import I, symbols
```

Comment:
I noticed that pylint doesn't seem to detect the unused imports here

`Dict, List, Tuple` are not used.

Reply:
Ah, this is because we disable https://docs.astral.sh/ruff/rules/unused-import/ (F401) in the `ruff.toml` file. I disabled it because I wanted to allow unused imports in init files, but it would be good to disallow them in other files. Will make an issue to fix this.